src/entities/entries.js

```diff
@@ -89,6 +89,15 @@ export const processEntryFilters = (filters, framework, geoOptions, hideMatrixFi
                 `${filterKey}__lt`,
                 getDateWithTimezone(encodeDate(endDate)),
             ]);
+        } else if (filterKey === 'verified') {
+            const isVerified = typeof filterOptions === 'string'
+                ? filterOptions === 'true'
+                : filterOptions;
+
+            result.push([
+                'verified',
+                isVerified,
+            ]);
```

Comment:
why are we doing this?

Reply:
New SelectInput doesn't support boolean. This is a hack.